frontend/lib/src/dataframes/Quiver.ts

```diff
@@ -123,68 +123,114 @@ type IntervalClosed = "left" | "right" | "both" | "neither"
 type IntervalType = `interval[${IntervalData}, ${IntervalClosed}]`
 
 // The frequency strings defined in pandas.
-// See: https://pandas.pydata.org/docs/user_guide/timeseries.html#dateoffset-objects
-type SupportedPandasOffsetType = "W" | "Q" | "D" | "H" | "T" | "S" | "L"
+// See: https://pandas.pydata.org/docs/user_guide/timeseries.html#period-aliases
+// Not supported: "N" (nanoseconds), "U" & "us" (microseconds), and "B" (business days).
+// Reason is that these types are not supported by moment.js, but also they are not
+// very commonly used in practice.
+type SupportedPandasOffsetType =
+  // yearly frequency:
+  | "A" // deprecated alias
+  | "Y"
+  // quarterly frequency:
+  | "Q"
+  // monthly frequency:
+  | "M"
+  // weekly frequency:
+  | "W"
+  // calendar day frequency:
+  | "D"
+  // hourly frequency:
+  | "H" // deprecated alias
+  | "h"
+  // minutely frequency
+  | "T" // deprecated alias
+  | "min"
+  // secondly frequency:
+  | "S" // deprecated alias
+  | "s"
+  // milliseconds frequency:
+  | "L" // deprecated alias
+  | "ms"
+
 type PeriodFrequency =
   | SupportedPandasOffsetType
   | `${SupportedPandasOffsetType}-${string}`
 type PeriodType = `period[${PeriodFrequency}]`
 
 const WEEKDAY_SHORT = ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"]
+const formatMs = (duration: number): string =>
```

Comment:
super nit: looks like this just adds some `duration` of seconds to 1970-01-01 and then formats it to that specific format. 
Should the function name be `addMsAndFormat` or something different?

Reply:
The value we have is just a millisecond number and we want to format that number. That it gets added to `1970` is more of a detail of the formatting itself (we want it to be formatted to a full date). So I think using `addMsAndFormat` would be slightly more confusing.